mkdocs.yml

```diff
@@ -70,8 +70,8 @@ pages:
     - What next: getting-started/what-next.md
 - Tutorials:
     - Introduction: tutorials/index.md
-    - Build a Handler: tutorials/handler.md
-    - Build a Generator: tutorials/generator.md
+    - Custom Commit Notifications: tutorials/custom-commit-notifications.md
+    - Teach your Team new Skills: tutorials/teach-atomist-new-skills.md
```

Comment:
I wanted to stress you improve your team, not Atomist. I thought our message was meant to say "Atomust majes you better", not "Make the Bot better".